OrientationAnalysis/test/ConvertOrientationsTest.cpp

```diff
@@ -55,20 +55,23 @@ void _make_code()
       if(inRep[i] == "qu")
       {
         std::cout << "  FromQuaternionFunctionType " << inRep[i] << "2" << outRep[o] << " = OrientationTransformation::" << inRep[i] << "2" << outRep[o] << "<QuaternionType, OutputType>;\n";
-        std::cout << "  parallelAlgorithm.execute(::FromQuaternion<float, FromQuaternionFunctionType, " << strides[i] << ", " << strides[o] << ">(inputDataArray, outputDataArray, " << inRep[i] << "2"
-                  << outRep[o] << ", QuaternionType::Order::VectorScalar));\n";
+        //   std::cout << "  ValidateInputDataFunctionType " << inRep[i] << "Check = " << names[i] << "Check<float>();" << std::endl;
```

Comment:
```suggestion
```
Not sure why this is commented out, remove if no longer necessary

Reply:
I left a comment in the code. This is code that generates code and I want to leave it in there. (until a better solution is figured out)